controller/array_action/array_mediator_svc.py

```diff
@@ -846,6 +846,26 @@ def _get_detailed_hosts_list_cmd(self, host_list):
             writer.write(LIST_HOSTS_CMD_FORMAT.format(HOST_ID=host.id))
         return writer.getvalue()
 
+    def get_host_by_name(self, host_name):
+        cli_host_by_name = self._get_cli_host_by_name(host_name, not_found_error=False)
+        if cli_host_by_name is None:
+            raise array_errors.HostNotFoundError(host_name)
+        cli_host_by_id = self._get_cli_host_by_id(cli_host_by_name.id)
+        if cli_host_by_id is None:
+            raise array_errors.HostNotFoundError(host_name)
```

Comment:
IMO it will be more readable to put it under `_get_cli_host_id_by_name` but do what you think is better (leave it that way or do my change)